.github/workflows/extended.yml

```diff
@@ -95,7 +106,7 @@ jobs:
       - name: Run tests
         run: |
           cd datafusion
-          cargo test  --profile ci --exclude datafusion-examples --exclude datafusion-benchmarks --exclude datafusion-sqllogictest --workspace --lib --tests --features=force_hash_collisions,avro,extended_tests
+          cargo test  --profile ci --exclude datafusion-examples --exclude datafusion-benchmarks --exclude datafusion-sqllogictest --workspace --lib --tests --features=force_hash_collisions,avro
```

Comment:
Why will we not run extended tests with hash collisions? We may lose some coverage here. If the only problematic thing is the memory-limited tests, we should place it under a separate flag and only skip those tests.

Reply:
I agree the fact that the flag `extended_tests` and the workflow is named `extended` is quite confusing

Maybe as a follow on PR we can rename the `extended_test` flag somthing different like `extra_tests` or `extended_suite` 🤔